tensorhive/models/CRUDModel.py

```diff
@@ -64,3 +67,30 @@ def get(cls, id):
     @classmethod
     def all(cls):
         return db_session.query(cls).all()
+
+    @classmethod
+    def _serialize(self, field):
+        if type(field) == datetime.datetime:
+            return DateUtils.stringify_datetime(field)
+        elif type(field) == bool:
+            return str(field)
```

Comment:
I'm not sure if I get your idea, as far I understand I've kept the previous way.

```
>>> a = True
>>> str(a)
'True'
```

Reply:
Previously using e.g. GET ​/restrictions we got `"isGlobal": true` in response, now we got `"isGlobal": "True"`
You can also see it by comparing response with Example response below

Reply:
Ah, now i see. In the Group model it was indeed raw boolean, and so it is in the corresponding API. I included the conversion to string because of the isCancelled field in Reservation model:

https://github.com/roscisz/TensorHive/blob/8f58f050f0112ef08a55c68e31c9645798ef4c95/tensorhive/models/Reservation.py#L159

This indeed should be unified. The isCancelled field is not included in API (why?), so I suppose that we should fall back to raw boolean as you suggested. Do you see any problems with that considering the above snippet?

Reply:
Reservation model was only place where we used str(boolean) - probably by mistake. We should use raw boolean everywhere.
Apparently, I forgot to update API, after adding isCancelled field in Reservation model. We should only update Reservation schema:
```
    Reservation:
      type: object
      required:
        ...
        - isCancelled
      properties:
        ...
        isCancelled:
          type: boolean
          example: false
```
isCancelled field cannot be updated by user, so ReservationForm and ReservationUpdateForm should not be updated.

Reply:
Thanks for your input! fixed in 945672a2ee2ef945fdbb3b6869b682d87f2b7cf2